src/navigations/HomeBottomTabs.js

```diff
@@ -131,7 +131,7 @@ function HomeBottomTabs() {
           tabBarLabel: () => menuIndicator(nav, route),
           tabBarIcon: ({focused, color}) => renderTabBarIcon(route, focused, color),
           tabBarActiveTintColor:
-            route.name === 'AnonymousChannelList' ? COLORS.anon_primary : COLORS.blueOnboarding
+            route.name === 'AnonymousChannelList' ? COLORS.anon_primary : COLORS.signed_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, this change also appears to be a color adjustment. Ensure that `COLORS.signed_primary` is defined and fits the UI design.
```diff
-    tabBarActiveTintColor: route.name === 'AnonymousChannelList' ? COLORS.anon_primary : COLORS.blueOnboarding
+    tabBarActiveTintColor: route.name === 'AnonymousChannelList' ? COLORS.anon_primary : COLORS.signed_primary
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->